File: myblog-server/src/main/java/com/site/blog/service/CategoryService.java
package com.site.blog.service;

import com.site.blog.model.entity.Category;
import com.baomidou.mybatisplus.extension.service.IService;

/**
 * <p>
 * 博客分类 服务类
 * </p>
 *
 */
public interface CategoryService extends IService<Category> {

    public boolean clearCategory(String id);

}

File: myblog-kt/src/main/kotlin/com/site/blog/service/CategoryService.kt
package com.site.blog.service

import com.baomidou.mybatisplus.extension.service.IService
import com.site.blog.model.entity.Category

/**
 *
 *
 * 博客分类 服务类
 *
 *
 */
interface CategoryService : IService<Category?> {
    fun clearCategory(id: String?): Boolean
}